spec/spec_helper.rb

```diff
@@ -90,6 +90,7 @@
   config.infer_spec_type_from_file_location!
 
   config.before { ActionMailer::Base.deliveries.clear }
+  config.filter_run_when_matching :focus
```

Comment:
Good call!

Reply:
Thanks! I will be sure to update the wiki too